### ISSUE TITLE
xlim: recompute the tick labels

### ISSUE THREAD

**Issue by janschulz** [yhat/ggplot#186]
currently setting limits to a axis will not recompute the axis labels/ticks. This is problematic if you use a very small excerpt of the former data: in the worst case no tick label is visible.

Problem can be seen here:
```python
p = ggplot(aes(x="price"), data=diamonds) + geom_histogram()
p = p + facet_wrap("cut", scales="free_y")
print(p + xlim(10,4900))
```
-> No ticklabel on x axis :-(

**Comment by eshilts** [comment]
This also fails for the y-axis whenever y-limits are set. It appears it is because the y-limits are calculated after y-labels in the ggplot.py module.

**Comment by glamp** [comment]
fixed in 0.9.3
